code/modules/reagents/reagent_containers/borghydro.dm

```diff
@@ -127,6 +127,10 @@ Borg Hypospray
 	if(empty)
 		usr << "<span class='warning'>It is currently empty! Allow some time for the internal syntheszier to produce more.</span>"
 
+/obj/item/weapon/reagent_containers/borghypo/hacked
+	icon_state = "borghypo_s"
+	reagent_ids = list ("facid", "mutetoxin", "cyanide", "sodium_thiopental", "heparin", "teslium")

```

Comment:
FAcid does Brute when Injected (I think around 18 per injection if I read it right.)
Cyanide is a standard toxin that lasts awhile with a small RNG chance to stun.
Sodium Thiopental is a Traitor Chem that puts you to sleep after awhile, while doing alot of stamina damage.
Heparin makes them bleed and does some brute damage over time.
Teslium randomly shocks them, ignoring any protection.
Overall, wouldn't mind switching out some of em if you have any suggestions.